### DISCUSSION TITLE
Can somebody help explain State Management to me?

### DISCUSSION THREAD

**Discussion by ekazda** [nuxt/nuxt#17724] [Questions]
I'm new to Nuxt and I'm excited to start learning Nuxt 3. I'm trying to follow the [this tutorial](https://developers.wpengine.com/blog/headless-wordpress-with-vue-and-nuxt-js) for how to build a headless WordPress site using Nuxt, but I'm getting hung up when they start **_Wiring Up The Data_** because `this.$state` no longer seems to be supported by Vuex. 

If anyone could take a moment to help explain how to resolve the `this.$state` issue, I would appreciate it.

**Comment by asonnleitner** [comment]
In the tutorial you mentioned they are using nuxt2. If you want to use vuex in nuxt3 you have to install vuex yourself or preferably use the new recommend state management library pinia as pointed out here https://v3.nuxtjs.org/migration/configuration#vuex
